package.json

```diff
@@ -44,27 +44,27 @@
     ]
   },
   "devDependencies": {
-    "@types/jest": "27.0.0",
+    "@types/jest": "29.2.4",
     "@types/node": "16.4.14",
-    "@typescript-eslint/eslint-plugin": "5.42.0",
-    "@typescript-eslint/parser": "5.42.0",
-    "eslint": "8.26.0",
+    "@typescript-eslint/eslint-plugin": "5.45.1",
+    "@typescript-eslint/parser": "5.45.1",
+    "eslint": "8.29.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-prettier": "4.2.1",
-    "husky": "7.0.4",
-    "jest": "27.0.6",
-    "lint-staged": "11.1.2",
-    "prettier": "2.7.1",
-    "pretty-quick": "3.1.1",
+    "husky": "8.0.2",
+    "jest": "29.3.1",
+    "lint-staged": "13.1.0",
+    "prettier": "2.8.0",
+    "pretty-quick": "3.1.3",
     "rimraf": "3.0.2",
-    "ts-jest": "27.0.4",
-    "typescript": "4.8.4"
+    "ts-jest": "29.0.3",
+    "typescript": "4.9.3"
   },
   "dependencies": {
-    "couchbase": "^3.2.6",
-    "jsonpath": "^1.1.1",
-    "lodash": "^4.17.21",
-    "uuid": "^8.3.2"
+    "couchbase": "4.2.0",
+    "jsonpath": "1.1.1",
+    "lodash": "4.17.21",
+    "uuid": "9.0.0"
```

Comment:
Did we decide to allow patch version discrepancy here: `^9.0.0`?

I don't recall where we landed on it. 

cc @thejcfactor

Reply:
I still need to talk w/ Brett about the previous reason Lino mentioned for pinning.  So, I think for this release we are okay to pin to a specific version.

Reply:
Sounds good!